src/applications/personalization/view-dependents/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "appName": "View Dependents",
+  "entryFile": "./view-dependents-entry.jsx",
+  "entryName": "disability-my-rated-disabilities",
```

Comment:
Should this entryName be something more aligned with view dependents? Maybe something like `dependents-view-dependents`?

Reply:
Yep, I updated this to dependents-view-dependents. As per your comment below Im not sure where this will end up falling in the IA so it may make sense to update this once that is figured out.